spec/rails_helper.rb

```diff
@@ -93,11 +93,6 @@
       GrdaWarehouse::WarehouseReports::ReportDefinition.maintain_report_definitions
     end
 
-    # load hmis forms if we're testing the driver
```

Comment:
I'm proposing to remove this, since I found it confusing that the database was seeded with forms when I ran a single test locally, but that same seeded data wasn't present when the whole test suite ran on CI.

Is this breaking anyone else's workflow? cc @ttoomey?

Reply:
Happy to remove it entirely if CI passes without it; I don't think it's breaking anyone's workflow. I'd made this conditional as a performance optimization (so iterating on a single spec was faster) but much better if we can just remove it.

Thanks!